.prerequisiteversions

```diff
@@ -0,0 +1 @@
+CMake=3.7.2
```

Comment:
Thanks Wes & Davis.  Let's name this ".cmakeversion" for now, and plan a unified name for all tools when bootstrap work is ready.